bin/reth/src/cli/mod.rs

```diff
@@ -158,6 +159,10 @@ pub struct Logs {
     /// The filter to use for logs written to the log file.
     #[arg(long = "log.filter", value_name = "FILTER", global = true, default_value = "error")]
     filter: String,
+
+    /// The flag to disable colour coding of console output.
```

Comment:
we can adapt the docs from with_ansi here:

> Sets whether or not the formatter emits ANSI terminal escape codes for colors and other text formatting.

perhaps we should also rename the flag to `no-colors`

defer to @onbjerg here

Reply:
Ok I wait @onbjerg, thanks.

Reply:
It's way more common to see a flag named `--color` that accepts `auto`, `never` or `always`.

The auto mode usually does some environment detection, but for now it's ok if it just behaves as `always` imo

I'd prefer that to this flag